foundry.toml

```
@@ -5,20 +5,12 @@ libs = ["lib"]
 solc = "0.8.24"
 optimizer = true
 optimizer_runs = 200
-verbosity = 3
-
-# See more config options https://github.com/foundry-rs/foundry/blob/master/crates/config/README.md#all-options
-
-[profile.test]
-verbosity = 3
-
+no_match_coverage = "(script|test|mocks)"
 fs_permissions = [
     { access = "read", path = "./out/" },
     { access = "read", path = "./script/input/" },
     { access = "read-write", path = "./script/output/" }
 ]
 
 [rpc_endpoints]
-mainnet = "${ETH_RPC_URL}"
-
-no_match_coverage = "test"
+mainnet = "${ETH_RPC_URL}"
```

Please [add a blank line](https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-a-source-file) to the end of the file.
BTW we should add this to an `.editorconfig` file.
```suggestion
mainnet = "${ETH_RPC_URL}"

```